libs/agno/tests/unit/reader/test_async_arxiv_reader.py

```diff
@@ -0,0 +1,214 @@
+from unittest.mock import Mock, patch
+
+import arxiv
+import pytest
+
+from agno.document.reader.arxiv_reader import ArxivReader
+
+
+@pytest.fixture
+def mock_arxiv_result():
```

Comment:
Might as well add the same tests for sync. Can be in the same file

Reply:
@dirkbrnd they already are there. sync and async in the same file in `unit/reader/test_arxiv_reader.py` and similarly for all readers